diff --git a/dist/esm/server/web/sandbox/context.js b/dist/esm/server/web/sandbox/context.js
index c6bb2e5dbed747a2be95882b1f9e4840f50d514b..4c31352f32fc3dff0fe12fea809a2d2d75602c82 100644
--- a/dist/esm/server/web/sandbox/context.js
+++ b/dist/esm/server/web/sandbox/context.js
@@ -199,10 +199,10 @@ export const requestStore = new AsyncLocalStorage();
     const { edgeFunctionEntry } = options;
     const wasm = await loadWasm(edgeFunctionEntry.wasm ?? []);
     const runtime = new EdgeRuntime({
-        codeGeneration: process.env.NODE_ENV !== "production" ? {
+        codeGeneration: {
             strings: true,
             wasm: true
-        } : undefined,
+        },
         extend: (context)=>{
             context.process = createProcessPolyfill(edgeFunctionEntry.env);
             Object.defineProperty(context, "require", {
diff --git a/dist/server/web/sandbox/context.js b/dist/server/web/sandbox/context.js
index 64ad82dac1482848422fdf82a9e17ec3d4686980..dd461d1951290a512909e110c976cf73a4f07ffd 100644
--- a/dist/server/web/sandbox/context.js
+++ b/dist/server/web/sandbox/context.js
@@ -225,10 +225,10 @@ const requestStore = new _async_hooks.AsyncLocalStorage();
     const { edgeFunctionEntry } = options;
     const wasm = await loadWasm(edgeFunctionEntry.wasm ?? []);
     const runtime = new _edgeruntime.EdgeRuntime({
-        codeGeneration: process.env.NODE_ENV !== "production" ? {
+        codeGeneration: {
             strings: true,
             wasm: true
-        } : undefined,
+        },
         extend: (context)=>{
             context.process = createProcessPolyfill(edgeFunctionEntry.env);
             Object.defineProperty(context, "require", {
